src/main/resources/templates/fragments/common.html

```diff
@@ -282,39 +282,39 @@ <h1 th:text="#{tla_Title}">Thesaurus Linguae Aegyptiae</h1>
       <!-- token wordclass -->
     <th:block th:fragment="token-word-class">
      <span th:if="${token.getPOS !=null}" th:utext="${token.getPOS}" 
-             class=".container-annotation.word-class"/>
```

Comment:
pattern ".container-annotation.word-class" invalid in class attribute => no style/class applies to this span